arch/x86/chunkset_avx2.c

```diff
@@ -68,6 +74,10 @@ static inline void chunkmemset_8(uint8_t *from, chunk_t *chunk) {
     *chunk = _mm256_set1_epi64x(tmp);
 }
 
+static inline void chunkmemset_16(uint8_t *from, chunk_t *chunk) {
```

Comment:
So we were missing one?

Reply:
Sort of.  It had been handled in the GET_CHUNK_MAG function but it was better to handle it here.